### DISCUSSION TITLE
Making your RPI 3B+ more reactive

### DISCUSSION THREAD

**Discussion by croisez** [mcguirepr89/BirdNET-Pi#797] [General]
I wanted to analyse a little bit the performances of my RPI3B+ running BirdNET-Pi.
I did not like the simple "top" application, natively present in Raspbian, so I found another tool, based on Python3:  bpytop.

To install it, simply type:
> pip3 install bpytop

And at the prompt, run it by:
> bpytop

The interresting part is that with bpytop, I remarked that my RPI had a full swap file amost permanently.
So I found a tutorial for oversizing it.
Here: https://pimylifeup.com/raspberry-pi-swap-file/
I set it to 200MB.
And I personaly find that with such value, my RPI3B+ is way more reactive now.

**Comment by lloydbayley** [comment]
Nice find! I have a spare Pi3+ running BirdNetPi testing the new model and I've just done it on there, so will test it out over the next day or two and see. I like the bpytop too. Very nice. 
For some reason, pip didn't install properly to the path so I had to run: ```/home/pi/.local/bin/bpytop``` instead.
Nevertheless, it does the trick! Will report back in due course!
